packages/rum/test/benchmarks/server.js

```diff
@@ -25,38 +25,44 @@
 
 const express = require('express')
 const path = require('path')
-const { elasticApmUrl, port } = require('./config')
-const { createReadStream } = require('fs')
+const { port, imageCount } = require('./config')
+const { getMinifiedApmBundle } = require('./analyzer')
 
 const pages = path.join(__dirname, 'pages')
-const dist = path.join(__dirname, '../../dist')
 
-function generateImageUrls(number) {
+function generateImageUrls() {
   const path = `http://localhost:${port}`
   const urls = []
-  for (let i = 0; i < number; i++) {
+  for (let i = 0; i < imageCount; i++) {
     urls.push(`${path}/images/${i}.png`)
   }
   return urls
 }
 
 /**
- * Avoid APM Script HTTP and Disk Cache in chrome
+ * Strip license and sourcemap url
  */
-function cacheBurstUrl(url) {
-  return url + '?' + Date.now()
+const APM_BUNDLE = getMinifiedApmBundle()
+  .replace(
+    '/*! For license information please see elastic-apm-rum.umd.min.js.LICENSE */',
+    ''
+  )
+  .replace('//# sourceMappingURL=elastic-apm-rum.umd.min.js.map', '')
```

Comment:
Why these comments need to be removed?

Reply:
to avoid browser from spending time in parsing this line and locating the sourcemaps.